btag.py

```diff
@@ -194,9 +194,13 @@ def getBTagShapeSF(self, df, src_name, scale_name, isCentral, return_variations)
             for scale in scale_list:
                 if source == central and scale != central:
                     continue
+
+                if source != central and scale == central:
+                    continue
```

Comment:
```suggestion
            for scale in getScales(source):
```
getScales is defined [here](https://github.com/cms-flaf/Corrections/blob/main/CorrectionsCore.py#L31)